manuscript/01-introduction.Rmd

```diff
@@ -32,4 +32,4 @@ You can either read the book from beginning to end or jump directly to the metho
 
 I hope you will enjoy the read!
 
-[^Hastie]: Friedman, Jerome, Trevor Hastie, and Robert Tibshirani. "The elements of statistical learning". hastie.su.domains/ElemStatLearn  (2009).
+[^Hastie]: Friedman, Jerome, Trevor Hastie, and Robert Tibshirani. "The elements of statistical learning". www.web.stanford.edu/~hastie/ElemStatLearn/  (2009).
```

Comment:
this is the old website. the current version has already the new one

Reply:
Do the other changes look good to you? I think I missed this particular one but the rest are most probably fine.